### DISCUSSION TITLE
v2.28.0 User account removal

### DISCUSSION THREAD

**Discussion by msp2842** [filebrowser/filebrowser#3091] [General]
All of my instances of file browser were updated and in doing so all of the changes within the app were reset. I looked at the release information and did not see any mention of that type of behavior. Anyone else run into that issue?

**Comment by aviadhaham** [comment]
Yeah, the user I created (not the default admin) was deleted, and I had to create it again. I am not sure what happened.

**Comment by niubility000** [comment]
We need to debug it from the middle of the Changelog, and test it one by one to locate the problem.
